src/renderer/components/dashboard/TokenStats.vue

```diff
@@ -140,11 +141,16 @@ export default {
       margin-left: $space-lg;
       font-family: Gilroy;
       font-size: toRem(20px);
+      position: absolute;
+      right: $space;
+      top: 0;
```

Comment:
Why are we using `position: absolute;` here? That should be last resort. Screenshot?